common/src/main/java/org/apache/celeborn/common/client/MasterNotLeaderException.java

```diff
@@ -19,22 +19,27 @@
 
 import java.io.IOException;
 
+import javax.annotation.Nullable;
+
+import org.apache.commons.lang3.StringUtils;
+
 public class MasterNotLeaderException extends IOException {
 
-  private final String currentPeer;
   private final String leaderPeer;
 
   public static final String LEADER_NOT_PRESENTED = "leader is not present";
 
-  public MasterNotLeaderException(String currentPeer, String suggestedLeaderPeer) {
+  public MasterNotLeaderException(
+      String currentPeer, String suggestedLeaderPeer, @Nullable Throwable cause) {
     super(
-        "Master:"
-            + currentPeer
-            + " is not the leader. Suggested leader is"
-            + " Master:"
-            + suggestedLeaderPeer
-            + ".");
-    this.currentPeer = currentPeer;
+        String.format(
+            "Master:%s is not the leader. Suggested leader is Master:%s.%s",
```

Comment:
` "Master:%s is not the leader. Suggested leader is Master:%s. %s"`
Add a blank space is better?

Reply:
@AngersZhuuuu, there is a blank space before `Exception`.